### PR TITLE
Add MAP Upgrade topic

### PR DESCRIPTION
**Pull request by smau-centreon**
## Description

Add MAP Upgrade topic

## Target version (i.e. version that this PR changes)

- [x] 22.04.x
- [x] 22.10.x
- [x] 23.04.x
- [x] 23.10.x
- [x] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Review by Jordan9131 — DISMISSED** [review]

**Review by Jordan9131 — CHANGES_REQUESTED** [review]

**Review by Jordan9131 — APPROVED** [review]

**Review by Jordan9131 — APPROVED** [review]

File: versioned_docs/version-22.10/graph-views/map-web-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+id: map-web-upgrade
+title: Upgrading MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> If you're updating to a new major or minor version (i.e:A.B.x with A or B that
+> changes) you need to contact our Support service to retrieve the new
+> repository.
```
Comment (cg-tw) [inline versioned_docs/version-22.10/graph-views/map-web-upgrade.md:10]:
```suggestion
> If you're updating to a new major version (i.e: A.B.x with A or B that
> changes) you need to install the new Business
> repository. You can find its address on the [support portal](https://support.centreon.com/hc/en-us/categories/10341239833105-Repositories).
```

File: versioned_docs/version-24.04/graph-views/map-web-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+id: map-web-upgrade
+title: Upgrading MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> If you're updating to a new major or minor version (i.e:A.B.x with A or B that
+> changes) you need to contact our Support service to retrieve the new
+> repository.
```
Comment (cg-tw) [inline versioned_docs/version-24.04/graph-views/map-web-upgrade.md:10]:
```suggestion
> If you're updating to a new major version (i.e: A.B.x with A or B that
> changes) you need to install the new Business
> repository. You can find its address on the [support portal](https://support.centreon.com/hc/en-us/categories/10341239833105-Repositories).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+id: map-web-upgrade
+title: Monter de version MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> Lorsque vous mettez à jour une nouvelle version majeure ou
+mineure (c'est à dire version A.B.x avec A ou B qui évolue), contactez
+le support pour récupérer l'adresse du nouveau dépôt.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-upgrade.md:10]:
```suggestion
> Lorsque vous mettez à jour une nouvelle version majeure (c'est à dire version A.B.x avec A ou B qui évolue), vous devez installer le nouveau dépôt Business. Vous pouvez trouver l'adresse du dépôt sur le [portail support Centreon](https://support.centreon.com/hc/fr/categories/10341239833105-D%C3%A9p%C3%B4ts).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
+---
+id: map-web-upgrade
+title: Monter de version MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> Lorsque vous mettez à jour une nouvelle version majeure ou
+mineure (c'est à dire version A.B.x avec A ou B qui évolue), contactez
+le support pour récupérer l'adresse du nouveau dépôt.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-upgrade.md:10]:
```suggestion
> Lorsque vous mettez à jour une nouvelle version majeure (c'est à dire version A.B.x avec A ou B qui évolue), vous devez installer le nouveau dépôt Business. Vous pouvez trouver l'adresse du dépôt sur le [portail support Centreon](https://support.centreon.com/hc/fr/categories/10341239833105-D%C3%A9p%C3%B4ts).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
+---
+id: map-web-upgrade
+title: Monter de version MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> Lorsque vous mettez à jour une nouvelle version majeure ou
+mineure (c'est à dire version A.B.x avec A ou B qui évolue), contactez
+le support pour récupérer l'adresse du nouveau dépôt.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-upgrade.md:10]:
```suggestion
> Lorsque vous mettez à jour une nouvelle version majeure (c'est à dire version A.B.x avec A ou B qui évolue), vous devez installer le nouveau dépôt Business. Vous pouvez trouver l'adresse du dépôt sur le [portail support Centreon](https://support.centreon.com/hc/fr/categories/10341239833105-D%C3%A9p%C3%B4ts).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/graph-views/map-web-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
+---
+id: map-web-upgrade
+title: Monter de version MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> Lorsque vous mettez à jour une nouvelle version majeure ou
+mineure (c'est à dire version A.B.x avec A ou B qui évolue), contactez
+le support pour récupérer l'adresse du nouveau dépôt.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/graph-views/map-web-upgrade.md:10]:
```suggestion
> Lorsque vous mettez à jour une nouvelle version majeure (c'est à dire version A.B.x avec A ou B qui évolue), vous devez installer le nouveau dépôt Business. Vous pouvez trouver l'adresse du dépôt sur le [portail support Centreon](https://support.centreon.com/hc/fr/categories/10341239833105-D%C3%A9p%C3%B4ts).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/graph-views/map-web-upgrade.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
+---
+id: map-web-upgrade
+title: Monter de version MAP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> Lorsque vous mettez à jour une nouvelle version majeure ou
+mineure (c'est à dire version A.B.x avec A ou B qui évolue), contactez
+le support pour récupérer l'adresse du nouveau dépôt.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/graph-views/map-web-upgrade.md:10]:
```suggestion
> Lorsque vous mettez à jour une nouvelle version majeure (c'est à dire version A.B.x avec A ou B qui évolue), vous devez installer le nouveau dépôt Business. Vous pouvez trouver l'adresse du dépôt sur le [portail support Centreon](https://support.centreon.com/hc/fr/categories/10341239833105-D%C3%A9p%C3%B4ts).
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by Jordan9131 — DISMISSED** [review]

**Review by Jordan9131 — DISMISSED** [review]

**Comment by cg-tw** [comment]
Approved because I have to since I requested changes, but I did not review the whole PR.

**Review by cg-tw — APPROVED** [review]

**Pull request merged**